bower.json

```diff
@@ -29,12 +29,12 @@
     "*.iws"
   ],
   "dependencies": {
-    "angular": "^1.2.0",
-    "angular-ui-router": "^0.2.8"
+    "angular": "~1.6.0",
```

Comment:
It should be `~1` to make it works with angular 1.5.x.
Angular 1.6.x has a breaking change (on `$http`) that can make migration difficult.